### ISSUE TITLE
Deck import error  Lord Jaraxxus

### ISSUE THREAD

**Issue by anonymous555** [HearthSim/Hearthstone-Deck-Tracker#4267]
Importing the following deck code misses Lord Jaraxus.  Using the latest version as of today.

```
### Control Warlock
# Class: Warlock
# Format: Standard
# Year of the Gryphon
#
# 2x (1) Armor Vendor
# 2x (1) Spirit Jailer
# 2x (2) Drain Soul
# 2x (2) Soul Shear
# 2x (3) Hysteria
# 2x (3) Luckysoul Hoarder
# 2x (3) School Spirits
# 1x (3) Tamsin Roame
# 2x (4) Cascading Disaster
# 2x (5) Ogremancer
# 2x (5) Siphon Soul
# 1x (6) Tickatus
# 1x (7) Soulciologist Malicia
# 2x (7) Strongman
# 2x (8) Twisting Nether
# 1x (9) Alexstrasza the Life-Binder
# 1x (9) Lord Jaraxxus
# 1x (10) Y'Shaarj, the Defiler
#
AAECAf0GBs/SA/zeA87hA/LtA7CKBIWgBAysywPM0gPN0gOO1APG3gPm4QP14wP44wOS5AOCoASDoATnoAQA


# To use this deck, copy it to your clipboard and create a new deck in Hearthstone
# Find this deck on https://hsreplay.net/decks/cAiJhTGootJ4uq5w4xjceh/
```

**Comment by anonymous555** [comment]
Here's another example deck

`AAECAf0GCM/SA/zeA87hA5boA/LtA6iKBLCKBIWgBAusywPM0gPN0gPG3gPm4QP14wP44wOS5AOCoASDoATnoAQA`

**Comment by mattestes** [comment]
Just wanted to add to this, that I'm having the same issue. Also, when I try to add Jaraxxus manually, it doesn't have the Core Set Hero card version of Lord Jaraxxus, just the Legacy minion version. It seems like it doesn't properly realize that the Hero card version exists.

**Comment by Agoriah** [comment]
This is the change that created the issue: 

> Forged in the Barrens 20.0.0.77662 (2021-03-25): Is now a hero card instead of a minion. Now grants 5 Armor. No longer sets your maximum Health to 15.

**Comment by anonymous555** [comment]
Any chance of getting this fixed? Seems like someone already provided a pull request with the fix
https://github.com/HearthSim/HearthDb/pull/24

**Comment by beheh** [comment]
This was fixed back in HearthSim/HearthDb#25.
